contracts/src/v0.8/vrf/dev/SubscriptionAPI.sol

```diff
@@ -204,35 +204,35 @@ abstract contract SubscriptionAPI is ConfirmedOwner, IERC677Receiver, IVRFSubscr
   }
 
   /*
-   * @notice Oracle withdraw LINK earned through fulfilling requests
+   * @notice withdraw LINK earned through fulfilling requests
    * @param recipient where to send the funds
    * @param amount amount to withdraw
    */
-  function oracleWithdraw(address recipient, uint96 amount) external nonReentrant {
+  function withdraw(address recipient, uint96 amount) external nonReentrant onlyOwner {
     if (address(LINK) == address(0)) {
       revert LinkNotSet();
     }
-    if (s_withdrawableTokens[msg.sender] < amount) {
+    if (s_withdrawableTokens < amount) {
```

Comment:
The behaviour change here is that
- Previously, the oracles could withdraw the earnings from VRFCoordinator
- Now, only the contract owner can withdraw the earnings
Is this correct?

Reply:
yeah. "oracles" could have been any identity previously. It didn't necessarily need to be the node fulfilling requests